modules/serverless-support.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// serverless-release-notes.adoc
+
+ [id="serverless-support_{context}"]
+= Getting support
+
+If you experience difficulty with a procedure described in this documentation, visit the link:https://access.redhat.com/support/offerings/techpreview/[Red Hat Customer Portal]. Through the customer portal, you can:
```

Comment:
But this actually links to the Tech Preview support statement on the customer portal. Which would be fine, except the list that follows explores what someone can do from the portal itself. Can it link to the portal directly and surface the TP support statement in another way?